packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -332,40 +337,15 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       return;
     }
 
-    // 3b: retrieve unmanged org source
+    // 3b: retrieve unmanaged org source
     await this.executeCommand(
       this.buildRetrieveOrgSourceCommand(response.data),
       { cwd: projectPath },
       cancellationTokenSource,
       cancellationToken
     );
 
-    // 4a: unzip retrieved source
```

Comment:
unzipping and converting metadata to force-app output-directory is taken care in step 3 where retrieving using manifest and output-dir is force-app by default.